src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -41,6 +41,14 @@ static constexpr std::wstring_view LightThemeValue{ L"light" };
 static constexpr std::wstring_view DarkThemeValue{ L"dark" };
 static constexpr std::wstring_view SystemThemeValue{ L"system" };
 
+static constexpr std::string_view DebugFeaturesKey{ "debugFeatures" };
+
+#ifdef _DEBUG
+static constexpr bool debugFeaturesDefault{ true };
+#else
+static constexpr bool debugFeaturesDefault{ false };
+#endif

```

Comment:
I hate to be _that_ guy. But could we add this to the docs?

Reply:
I'm more thinking _this is a feature we tell people to enable, then they turn it off_. I don't want to put it in the docs because adding the _key_ versus adding _features that depend_ on the key are two separate operations. We're practically guaranteed to only update the docs once.

Reply:
I'm (almost?) to the point where I'd rather not have a debugging feature that'll help us if it means writing up a bunch of user-facing documentation, fwiw. The docs can come after, if we deem them necessary and if we do start using this in field diagnostics.

Reply:
This is really cool!

Reply:
I’m glad you think so!